foundation/foundation-mda/src/main/resources/templates/deployment/model-training-api/model-training-api.values.yaml.vm

```diff
@@ -21,8 +21,8 @@ service:
 configMap:
   trainingApiConfig: |
     project_name=${projectName}
-    docker_registry=ghcr.io/
-    image_prefix=boozallen/
+    docker_registry=${dockerProjectRepositoryUrl}
+    image_prefix=
```

Comment:
I: This chart runs a project-specific image, and so needs to run with the project-specific settings.